{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/authfoundation\/oauth2error\/failurereason"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OAuth2Error\/failureReason","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"LocalizedError.failureReason"},{"type":"text","text":"."}],"kind":"symbol","metadata":{"role":"symbol","title":"failureReason","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"symbolKind":"property","externalID":"s:10Foundation14LocalizedErrorPAAE13failureReasonSSSgvp::SYNTHESIZED::s:14AuthFoundation11OAuth2ErrorO","extendedModule":"Foundation","modules":[{"name":"AuthFoundation"}]},"hierarchy":{"paths":[["doc:\/\/AuthFoundation\/documentation\/AuthFoundation","doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OAuth2Error","doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OAuth2Error\/LocalizedError-Implementations"]]},"references":{"doc://AuthFoundation/documentation/AuthFoundation/OAuth2Error/LocalizedError-Implementations":{"role":"collectionGroup","title":"LocalizedError Implementations","abstract":[],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OAuth2Error\/LocalizedError-Implementations","kind":"article","type":"topic","url":"\/documentation\/authfoundation\/oauth2error\/localizederror-implementations"},"doc://AuthFoundation/documentation/AuthFoundation":{"role":"collection","title":"AuthFoundation","abstract":[{"type":"text","text":"Library that provides foundational features used by Oktaâ€™s authentication libraries, as well as APIs used to work with tokens and user credentials."}],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation"},"doc://AuthFoundation/documentation/AuthFoundation/OAuth2Error/failureReason":{"role":"symbol","title":"failureReason","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OAuth2Error\/failureReason","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation\/oauth2error\/failurereason"},"doc://AuthFoundation/documentation/AuthFoundation/OAuth2Error":{"role":"symbol","title":"OAuth2Error","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"OAuth2Error"}],"abstract":[],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OAuth2Error","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"OAuth2Error"}],"url":"\/documentation\/authfoundation\/oauth2error"}}}